src/torchaudio/functional/filtering.py

```diff
@@ -998,7 +998,7 @@ def _lfilter_core(
 
 
 def lfilter(waveform: Tensor, a_coeffs: Tensor, b_coeffs: Tensor, clamp: bool = True, batching: bool = True) -> Tensor:
-    r"""Perform an IIR filter by evaluating difference equation.
+    r"""Perform an IIR filter by evaluating difference equation, using differentiable implementation developed independently by *Yu et al.* :cite:`ismir_YuF23` and *Forgione et al.* :cite:`forgione2021dynonet`.
```

Comment:
`flake8` is freaking out. 

https://github.com/pytorch/audio/actions/runs/7630525757/job/20786415251?pr=3734

Could you make this into a separate line?

Reply:
Fixed.